pxr/usd/sdf/schema.cpp

```diff
@@ -872,6 +872,8 @@ SdfSchemaBase::_RegisterStandardFields()
                        SdfMetadataDisplayGroupTokens->core)
         .MetadataField(SdfFieldKeys->DisplayGroupOrder,
                        SdfMetadataDisplayGroupTokens->core)
+        .MetadataField(SdfFieldKeys->DisplayName,
```

Comment:
We'll add this to field keys in wrapPrimSpec.cpp as well

Reply:
added on the last commit